core/src/test/java/de/evoila/cf/broker/service/impl/AsyncBindingServiceImplTest/AsyncCreateBindingTest.java

```diff
@@ -0,0 +1,73 @@
+package de.evoila.cf.broker.service.impl.AsyncBindingServiceImplTest;
+
+import de.evoila.cf.broker.exception.*;
+import de.evoila.cf.broker.model.JobProgress;
+import de.evoila.cf.broker.model.ServiceInstanceBindingRequest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+
+import static org.mockito.Mockito.*;
+
+class AsyncCreateBindingTest extends BaseTest {
+
+    @Mock
+    ServiceInstanceBindingRequest serviceInstanceBindingRequest;
+
+    private final static String DESCRIPTION = "Creating binding..";
+
+    @Test
+    @DisplayName("Should log exception, when jobStart(...) throws ServiceBroker Exception")
+    void startJobThrowsServiceBrokerException() throws ServiceBrokerException {
+        ServiceBrokerException serviceBrokerException = new ServiceBrokerException("Test");
+        mockStartJobThrowsException(JobProgress.BIND, DESCRIPTION, serviceBrokerException);
+        runCreateBinding();
+    }
+
+    @Test
+    @DisplayName("Should log exception, when jobStart(...), throws RuntimeException")
+    void startJobThrowsRuntimeException() throws ServiceBrokerException {
+        RuntimeException runtimeException = new RuntimeException("Test");
+        mockStartJobThrowsException(JobProgress.BIND, DESCRIPTION, runtimeException);
+        runCreateBinding();
+        verify(jobProgressService, times(1))
+                .failJob(JOB_PROGRESS_ID, "Internal error during instance binding creation, please contact our support.");
+    }
+
+    @Test
+    @DisplayName("Should log exception, when createBinding(...) throws ServiceBrokerException")
+    void createBindingThrowsServiceBrokerException() throws ServiceBrokerException, InvalidParametersException, PlatformException {
+        mockStartJobReturnsJobProgress(JobProgress.BIND, DESCRIPTION);
+        when(bindingService.createBinding(SERVICE_BINDING_ID, serviceInstance, serviceInstanceBindingRequest, plan))
```

Comment:
Maybe write helper method mockCreateBindingThrowsException (same as for startJob)?

Reply:
fixed